jobs/availability.rb

```diff
@@ -57,10 +57,12 @@ def getUptimeMetricsFromPingdom(checkId, apiKey, user, password)
   performCheckAndSendEventToWidgets('change-credentials', 'ft-memb-user-cred-svc-at-elb-eu-946129326.eu-west-1.elb.amazonaws.com', '/tests/change-credentials-critical', false)
   performCheckAndSendEventToWidgets('reset-password', 'ft-memb-user-cred-svc-at-elb-eu-946129326.eu-west-1.elb.amazonaws.com', '/tests/reset-password-critical', false)
   performCheckAndSendEventToWidgets('validate-session', 'ft-memb-session-api-at-elb-p-301208839.eu-west-1.elb.amazonaws.com', '/tests/critical-validate', false)
+  performCheckAndSendEventToWidgets('get-user-products', 'usr-product-svc-lb-eu-west-1-prod.memb.ft.com', '/tests/get-products-critical', false)
```

Comment:
After a discussion with the team, it has been approved that instead of provisioning nodes for the acceptance tests, we should just deploy it to Heroku instead. So the domain value must be changed to point to Heroku, please don't approve this PR unless this is changed.